freqtrade/configuration.py

```diff
@@ -375,22 +376,41 @@ def get_config(self) -> Dict[str, Any]:
 
         return self.config
 
-    def check_exchange(self, config: Dict[str, Any]) -> bool:
+    def check_exchange(self, config: Dict[str, Any], check_for_bad: bool = True) -> bool:
         """
         Check if the exchange name in the config file is supported by Freqtrade
-        :return: True or raised an exception if the exchange if not supported
+        :param check_for_bad: if True, check the exchange against the list of known 'bad'
+                              exchanges
+        :return: False if exchange is 'bad', i.e. is known to work with the bot with
+                 critical issues or does not work at all, crashes, etc. True otherwise.
+                 raises an exception if the exchange if not supported by ccxt
+                 and thus is not known for the Freqtrade at all.
         """
-        exchange = config.get('exchange', {}).get('name').lower()
-        if not is_exchange_supported(exchange):
-
-            exception_msg = f'Exchange "{exchange}" not supported.\n' \
-                            f'The following exchanges are supported: ' \
-                            f'{", ".join(supported_exchanges())}'
+        logger.info("Checking exchange...")
 
-            logger.critical(exception_msg)
+        exchange = config.get('exchange', {}).get('name').lower()
+        if not is_exchange_known(exchange):
             raise OperationalException(
-                exception_msg
+                    f'Exchange "{exchange}" is not supported by ccxt '
+                    f'and not known for the bot.\n'
+                    f'The following exchanges are supported by ccxt: '
+                    f'{", ".join(known_exchanges())}'
             )
 
-        logger.debug('Exchange "%s" supported', exchange)
+        logger.info(f'Exchange "{exchange}" is supported by ccxt and known for the bot.')
+
+        if is_exchange_officially_supported(exchange):
+            logger.info(f'Exchange "{exchange}" is officially supported '
+                        f'by the Freqtrade development team.')
+        else:
+            logger.warning(f'Exchange "{exchange}" is not officially supported '
+                           f'by the Freqtrade development team. '
+                           f'It may work with serious issues or not work at all. '
+                           f'Use it at your own discretion.')
```

Comment:
I would word this message in a positive form. This makes it look like it is almost guaranteed to not work.

something like the following, maybe?

```
Exchange  ... is not officially supported yet.
It may work flawlessly (please report back), or have serious issues.
Use it at your own discretion.
```

Reply:
Agree about more positive form, consider my messages just as temp. examples created during development...

@xmatthias that's simply a question of wording, we'll discuss all the nuances of the phrases here in many comments, could you better change wording right in the branch instead? That would be more efficient...

Reply:
The main idea here was to avoid calling exchanges supported by ccxt simply as 'supported' because the only supported exchanges for freqtrade are binance and bittrex...